frontend/lib/src/dataframes/Quiver.ts

```diff
@@ -176,7 +187,7 @@ export class Quiver {
 
   /** List of column types for every index- & data-column. */
   public get columnTypes(): ArrowType[] {
-    return this._pandasIndexColumnTypes.concat(this._dataColumnTypes)
+    return this._columnTypes
```

Comment:
Small optimization since the `columnTypes` can get called a lot during data viewing.